app/components/GlobalError/GlobalError.js

```diff
@@ -21,6 +21,20 @@ class GlobalError extends React.Component {
   render() {
     const { error, clearError } = this.props
 
+    let errors = []
+
+    if (error) {
+      if (Array.isArray(error)) {
+        errors = error
+      } else if (typeof error === 'string') {
+        errors.push(error)
+      } else if (typeof error === 'object' && !Array.isArray(error)) {
```

Comment:
`!Array.isArray(error)` looks like a redundant check to me because we are under `else` from  `if (Array.isArray(error))`

Reply:
Good catch, thanks @korhaliv - I've updated that.